docs/SUMMARY.md

```diff
@@ -33,6 +33,7 @@
 ## Modding and Development
 
 * [Modding](modding/README.md)
+  * [Modding Tools](modding/tools.md)
```

Comment:
Probably best to move this section further down, at least below "Getting Started".